.gitignore

```diff
@@ -3,19 +3,76 @@
 /RemoteSystemsTempFiles/
 /bin/
 /.settings/
+*/.settings/
```

Comment:
Does `**/.settings/` not cover them all?

Reply:
It does, pushed that and cleaned up any duplicates.